lib/puppet-lint/plugins/check_unsafe_interpolations.rb

```diff
@@ -1,6 +1,8 @@
 PuppetLint.new_check(:check_unsafe_interpolations) do
   COMMANDS = Array['command', 'onlyif', 'unless']
   INTERPOLATED_STRINGS = Array[:DQPRE, :DQMID]
+  RESOURCE_TYPES = Array['package', 'file', 'service', 'notify', 'exec', 'cron', 'user', 'group']
```

Comment:
Why do we need to know about other resource types?

Reply:
it's a good question... I suppose I didn't think much about it because we filter for `exec` resources after we get the title tokens... but makes sense to do that here.

Should I not return the resource_type in get_title_tokens then since it will always be exec?